### PR TITLE
tools: add vscode workspace configuration

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Adds a VSCode workspace configuration file to support opening Clutch backend and frontend simultaneously in the same vscode window.

### Testing Performed

manual

### CODE REVIEW

File: .vscode/clutch.code-workspace [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		{
+			"path": "../frontend"
+		},
+		{
+			"path": "../tools"
+		},
+		{
+      "name": "ci",
+      "path": "../.github"
+    }
```
Comment (mcutalo88) [inline .vscode/clutch.code-workspace:21]: nit: fix formatting

**Comment by danielhochman** [comment]
weird, vscode works for me with no config

**Comment by mcutalo88** [comment]
Interesting it has never worked for me on OSX

**Review by mcutalo88 — APPROVED** [review]

**Comment by dschaller** [comment]
Certain features of VSCode will work but some do not. @danielhochman can you open up a test file and see if typescript can resolve global types for jest? That didn't work for me on mac

**Pull request merged**